docs/blog-articles/02-bpm-ideal-zouk-brasileiro-pt.md

```
@@ -0,0 +1,75 @@
+# ARTIGO 02 — PORTUGUÊS
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)